.ci/README.md

```diff
@@ -131,7 +131,7 @@ The CI can run on a local VM using the following instructions:
     $ sudo -H -u $USER bash -c 'echo export USE_PODMAN=true >> ~/.bashrc'
     $ sudo -H -u $USER bash -c 'echo cd ~/go/src/github.com/kata-containers/tests >> ~/.bashrc'
     $ sudo -H -u $USER bash -c 'echo source .ci/ci_job_flags.sh >> ~/.bashrc'
-    $ (! mount -t cgroup2 | grep "/sys/fs/cgroup\ ") && sudo grubby --update-kernel=ALL --args="systemd.unified_cgroup_hierarchy=0"
+    $ (mount -t cgroup2 | grep "/sys/fs/cgroup\ ") && sudo grubby --update-kernel=ALL --args="systemd.unified_cgroup_hierarchy=0"
```

Comment:
It's not strictly part of this PR, but ideally we'd also remove the match on the separator. Further, the path isn't guaranteed to match _only_ for `/sys/fs/cgroup` (Something like `/foo/sys/fs/cgroup` would also match), so I'm wondering about something like the following, which is robust:

```bash
mount -t cgroup2 | awk '{print $3}' | grep -E "^/sys/fs/cgroup\>"
```

Reply:
The idea was more about not to match "/sys/fs/cgroup/unified" because AFAIU when you have both v1 and v2, and v2 is disabled it usually mounted under /unified, and v1 is used is instead under /sys/fs/cgroup.
TBH I don't like to check it like that, but i couldn't find a better way, i'm open for ideas

Reply:
@jodh-intel WDYT^?

Reply:
How about:

```suggestion
    $ mount -t cgroup2 | grep -q "/sys/fs/cgroup[^/]" && sudo grubby --update-kernel=ALL --args="systemd.unified_cgroup_hierarchy=0"
```